packages/cli/src/lib/webdriver.test.ts

```diff
@@ -42,8 +42,8 @@ describe('startDriver', () => {
     browser = 'chrome-headless';
     driver = await startDriver(config);
     const capabilities = await driver.getCapabilities();
-
-    assert.equal(capabilities.get('browserName'), 'chrome');
+    // https://github.com/seleniumbase/SeleniumBase/issues/2343
+    assert.equal(capabilities.get('browserName'), 'chrome-headless-shell');
```

Comment:
I would do a similar check to what [selenium's expected code update is doing](https://github.com/seleniumbase/SeleniumBase/issues/2343#issuecomment-1846526333)

```js
assert.isTrue('chrome' in capabilities.get('browserName'))
```